randovania/games/prime_hunters/generator/base_patches_factory.py

```diff
@@ -0,0 +1,35 @@
+from __future__ import annotations
+
+from copy import copy
+from typing import TYPE_CHECKING
+
+from randovania.games.prime_hunters.layout import HuntersConfiguration
+from randovania.games.prime_hunters.layout.force_field_configuration import LayoutForceFieldRequirement
+from randovania.generator.base_patches_factory import BasePatchesFactory, MissingRng
+
+if TYPE_CHECKING:
+    from random import Random
+
+    from randovania.game_description.game_description import GameDescription
+
+
+class HuntersBasePatchesFactory(BasePatchesFactory[HuntersConfiguration]):
+    def create_game_specific(self, configuration: HuntersConfiguration, game: GameDescription, rng: Random) -> dict:
```

Comment:
Make this a typed dict

Reply:
Iirc we discussed that in the discord two weeks ago and because of the base class, you cannot use a `TypedDict`. Can be refactored later and I wouldn't mix a refactor in base classes with a PR which wants to add a game.

Reply:
I definitely tried it before and had issues so I reverted it

Reply:
👍